tasks/system_probe.py

```diff
@@ -660,7 +678,13 @@ def build_network_ebpf_files(ctx, build_dir, parallel_build=True, kernel_release
         (p, debug) = flavor[i]
         promises_link.append(build_network_ebpf_link_file(ctx, parallel_build, build_dir, p, debug, network_flags))
 
-    for promise in promises_link:
+    promises_strip = []
```

Comment:
Can we make the stripping optional and off by default? I think we only need to run it during the CI so the packaged build is smaller. I think we want to retain it for local dev, because I believe it can have value for debugging.

Reply:
Yes I can do that